src/AzureIoTHub.Portal.Tests.Unit/Client/Pages/AWS/CreateThingTypePageTests.cs

```diff
@@ -0,0 +1,197 @@
+            var saveButton = cut.WaitForElement("#SaveButton");
+            var addPropertyButton = cut.WaitForElement("#addTagButton");
+
+            cut.WaitForElement($"#{nameof(ThingTypeDetails.ThingTypeName)}").Change(Guid.NewGuid().ToString());
+            cut.WaitForElement($"#{nameof(ThingTypeDetails.ThingTypeDescription)}").Change(Guid.NewGuid().ToString());
+
+            addPropertyButton.Click();
+
+            cut.WaitForElement($"#tag- #{nameof(ThingTypeTagDetails.Key)}").Change(key);
+
+            var propertyCssSelector = $"#tag-{key}";
+
+            cut.WaitForElement($"{propertyCssSelector} #{nameof(ThingTypeTagDetails.Key)}").Change(key);
+            cut.WaitForElement($"{propertyCssSelector} #{nameof(ThingTypeTagDetails.Value)}").Change(value);
+
+            cut.WaitForAssertion(() => Assert.AreEqual(1, cut.FindAll("#deleteTagButton").Count));
+
+            saveButton.Click();
+
+            // Assert
+            cut.WaitForAssertion(() => Services.GetRequiredService<FakeNavigationManager>().Uri.Should().EndWith("/device-models"));
+            cut.WaitForAssertion(() => MockRepository.VerifyAll());
+        }
+
+        [Test]
+        public void ClickOnAddSearchableAttributeShouldAddNewSearchableAttr()
+        {
+            // Arrange
+            var name = Guid.NewGuid().ToString();
+
+            _ = Services.AddSingleton(new PortalSettings { CloudProvider = "AWS" });
+
+            _ = this.mockThingTypeClientService.Setup(service =>
+                    service.CreateThingType(It.IsAny<ThingTypeDetails>()))
+                .Returns(Task.CompletedTask);
+
+            _ = this.mockSnackbarService.Setup(c => c.Add(It.IsAny<string>(), Severity.Success, It.IsAny<Action<SnackbarOptions>>(), It.IsAny<string>())).Returns((Snackbar)null);
+
+            // Act
+            var cut = RenderComponent<CreateDeviceModelPage>();
+            var saveButton = cut.WaitForElement("#SaveButton");
+            var addPropertyButton = cut.WaitForElement("#addSearchableAttButton");
+
+            cut.WaitForElement($"#{nameof(ThingTypeDetails.ThingTypeName)}").Change(Guid.NewGuid().ToString());
+            cut.WaitForElement($"#{nameof(ThingTypeDetails.ThingTypeDescription)}").Change(Guid.NewGuid().ToString());
+
+            addPropertyButton.Click();
+
+            cut.WaitForElement($"#search- #{nameof(ThingTypeSearchableAttDto.Name)}").Change(name);
+
+            var propertyCssSelector = $"#search-{name}";
+
+            cut.WaitForElement($"{propertyCssSelector} #{nameof(ThingTypeSearchableAttDto.Name)}").Change(name);
+
+            cut.WaitForAssertion(() => Assert.AreEqual(1, cut.FindAll("#deleteSearchableButton").Count));
+
+            saveButton.Click();
+
+            // Assert
+            cut.WaitForAssertion(() => Services.GetRequiredService<FakeNavigationManager>().Uri.Should().EndWith("/device-models"));
+            cut.WaitForAssertion(() => MockRepository.VerifyAll());
+        }
+    }
+}
```

Comment:
Move unit tests in CreateThingTypePageTests to CreateDeviceModelPage. No need to create a new dedicated unit test class